common/app/conf/switches/FeatureSwitches.scala

```diff
@@ -530,7 +530,7 @@ trait FeatureSwitches {
     SwitchGroup.Feature,
     "we-are-hiring",
     "When ON, hiring messages will appear in browser console and HTML source",
-    owners = Seq(Owner.withGithub("siadcock")),
+    owners = Seq(Owner.withName("dotcom/platform")),
```

Comment:
I think in the past we have used `dotcom.platform` https://github.com/guardian/frontend/blob/f347296b927fc0ee74ff4413b4cbb9f4e948bf51/common/app/mvt/MultiVariateTesting.scala#L94